bindings/rust/s2n-tls-tokio/tests/common/mod.rs

```diff
@@ -50,6 +63,20 @@ pub fn server_config() -> Result<config::Builder, Error> {
     Ok(builder)
 }
 
+pub fn client_config_tls12() -> Result<config::Builder, Error> {
```

Comment:
Being a test we should be explicit rather than falling back on default behavior. I think it would be better for readibility. You could do it in a separate PR to avoid the PR bloat.